bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/discovery/AndroidDebugBridgeDiscoveryService.java

```diff
@@ -61,9 +67,13 @@
     private boolean discoveryRunning = false;
 
     @Activate
-    public AndroidDebugBridgeDiscoveryService(@Reference ConfigurationAdmin admin) {
+    public AndroidDebugBridgeDiscoveryService(final @Reference ConfigurationAdmin admin, //
+            final @Reference TranslationProvider i18nProvider, //
+            final @Reference LocaleProvider localeProvide) {
```

Comment:
These 2 references are added but not absolutely required?

Reply:
What is your proposal here. Shall I remove them?

Reply:
If you agree with me that they are useless, yes I would suggest to not add them.

Reply:
I removed the service references.